audio_sync/cli.py

```diff
@@ -126,6 +135,36 @@ def _PlotResults(
   print(output)
 
 
+def _PlotAsciiGraph(
+    latencies, start_time, dots_per_usec=0.01, latency_threshold_secs=0.001):
+  """Plots all latencies with timestamp in an ASCII timeline."""
```

Comment:
Please, document parameters and sanity-check args (e.g., `dots_per_usec` is > 0).

Reply:
Done.